onnx/defs/schema.cc

```diff
@@ -775,6 +823,19 @@ OpName_Domain_Version_Schema_Map& OpSchemaRegistry::map() {
 #endif
 
       RegisterOnnxOperatorSetSchema();
+	  
+	  // Verify functionbody in every node
+	  // As we already has the mutex now, we
+	  // can directly access the registry map
+      auto& m = GetMapWithoutEnsuringRegistration();
+      for (auto& opset_entry_ : m) {
+        for (auto& op_versions_entry_ : opset_entry_.second) {
+          for (auto& op_entry_ : op_versions_entry_.second) {
+            VerifyFunction(op_entry_.second, m);
```

Comment:
Hi, why not do the call to VerifyFunction as an extra step in "Finalize" (around line 690 in old file and line 740 in your version)? I think that will be better.